roles/comanage_enable/templates/apache.conf.j2

```diff
@@ -77,6 +85,14 @@
         MellonSessionLength 86400
         MellonNoCookieErrorPage "{{ comanage_full_hostname }}/no_cookie.html"
 
+{#
+        # We need an explicit metadata file to correct the HTTP->HTTPS
+        # change when we go past the LB
+        # If we don't specifiy the metadata file, mod_mellon will create
+        # one by itself, but use the HTTP protocol, because that is what
+        # we use in the backend
+#}
+        MellonSPMetadataFile /etc/apache2/mellon/sp-metadata.xml
```

Comment:
I've just tested this: only the fix to ServerName seems necessary, and we don't need to specify the SPMetaDataFile.  

In fact, I think it's an oversight that we're shipping the metadatafile at all; I think I forgot to remove it when I switched mellon to automatically generated metadata.